fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/postgresql/PostgreSqlResourceDAO.java

```diff
@@ -78,13 +82,104 @@ public PostgreSqlResourceDAO(TransactionSynchronizationRegistry trxSynchRegistry
      * @throws FHIRPersistenceDBConnectException
      * @throws FHIRPersistenceVersionIdMismatchException
```

Comment:
`throws` don't match.

Reply:
yeah, good catch, better to be consistent with the codes, even though the exceptions are detailed sub-classes of FHIRPersistenceException from inside the function.